### ISSUE TITLE
Use part of the dataset as a test set for accuracy

### ISSUE THREAD

**Issue by kris701** [jamadaha/P5#132]
**Is your feature request related to a problem? Please describe.**
Currently, all data is used as the training set. Some of the data could be used for testing.

**Describe the solution you'd like**
Make some of the train set available as test set
